packages/openapi-v3-types/test/unit/openapi-v3-spec-types.unit.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3-types
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {
+  ExampleObject,
+  ReferenceObject,
+  DiscriminatorObject,
+  XMLObject,
+  ExternalDocumentationObject,
+  ISpecificationExtension,
+  createEmptyApiSpec,
+  OpenAPIObject,
+} from '../..';
+
+describe('openapi-v3-types unit tests', () => {
```

Comment:
Looking at the tests written below, wouldn't it be more accurate to describe this `describe` block as `createEmptyApiSpec unit tests`?